packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -221,10 +224,10 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
   }, [setIsPressed])
 
   const handleLongPress = useCallback(() => {
-    if (message.status !== Status.ERROR) {
+    if (message && message.status !== Status.ERROR) {
       onLongPress?.(message.message_id)
     }
-  }, [message.message_id, message.status, onLongPress])
+  }, [message, onLongPress])
```

Comment:
nit: could keep the deps `message?.status` and `messageid` to prevent unnecessary rerenders